package.json

```diff
@@ -151,6 +151,21 @@
                     "type": "boolean",
                     "description": "Use JavaScript source maps (if they exist)",
                     "default": true
+                },
+                "vscode-edge-devtools.localization": {
+                    "type": "string",
+                    "enum": [
+                        "Enabled",
+                        "Disabled",
+                        "qps-ploc"
+                    ],
+                    "enumDescriptions": [
+                        "Elements for Edge will match the selected locale for VS Code",

```

Comment:
Needs to be Elements for *Microsoft* Edge

Reply:
done.